framework/Element.js

```diff
@@ -211,4 +211,13 @@ export default class Element {
         const elementHandle = await this.wait();
         await elementHandle.setInputFiles(filePath);
     }
+
+    async downloadFile(filePath, isAbsolutePath) {
+        if (!isAbsolutePath) filePath = process.cwd() + filePath;
```

Comment:
Maybe path.isAbsolute(path) can be used here, instead of passing a parameter? If not I suggest adding default value true for isAbsolutePath parameter.

Reply:
Changed to the method, removed method parameter